### ISSUE TITLE
RemoteVideoState is not getting updated

### ISSUE THREAD

**Issue by Meherdeep** [AgoraIO-Extensions/Agora-Flutter-SDK#318]
In the remoteVideoStateChanged() callback when a user updates it's video state from a browser the `Decoding` state is not triggered. 

**To Reproduce**
1. Run an Agora app on web and disable your video
2. In the Agora logs you will see that `Stopped` event is triggered which is perfect. 
3. But when you enable the video from the browser the `Decoding` event is not triggered.

**Expected behavior**
Reflect the change of state on the mobile device as well. 

**Desktop:**
 - OS: MacOS
 - Browser Safari
 - Version 11.3

**Smartphone:**
 - Device: Pixel 5
 - OS: Android

**Comment by plutoless** [comment]
can you see the video from web start playing again on flutter app when web enables video?

**Comment by Meherdeep** [comment]
Yes, the problem occurs just with the callback. It returns all the value except for `Decoding`.

**Comment by plutoless** [comment]
@Meherdeep is the `Stopped` event trigged on native side?

**Comment by Meherdeep** [comment]
Yes, it does. I have attached an image of the logs that I am generating. Over here you will see that when the remote user video started it returned `Starting` and `Decoding` state. And then why I disabled the video from the web client it returned the `Stopped` state as expected. But no further state was returned when I enabled the camera from the web.  

![image](https://user-images.githubusercontent.com/30744897/118391011-ce046c00-b64f-11eb-8d87-f8be70e28829.png)
